Header header
float64 x
float64 y
float64 theta
string algorithm_type